lib/puppet/provider/package/pip.rb

```diff
@@ -31,7 +31,11 @@ def self.instances
     packages = []
     pip_cmd = self.pip_cmd
     return [] unless pip_cmd
-    execpipe "#{pip_cmd} freeze" do |process|
+    command = ["#{pip_cmd}", 'freeze']
```

Comment:
Don't need to interpolate a string here. Just use `command = [pip_cmd, 'freeze']`